impl/src/main/java/org/ehcache/config/CacheConfigurationBuilder.java

```diff
@@ -136,4 +142,59 @@ private CacheConfigurationBuilder(CacheConfigurationBuilder<? super K, ? super V
   public boolean hasDefaultExpiry() {
     return expiry == null;
   }
+
+  public CacheConfigurationBuilder<K, V> withEvictionVeto(EvictionVeto<K, V> evictionVeto) {
```

Comment:
At the same time, not sure I am convinced that adding the `with` prefix gains us anything. What do other think? @chrisdennis @cljohnso @AbfrmBlr